src/Aspire.Hosting/DistributedApplicationLifecycle.cs

```diff
@@ -17,6 +21,11 @@ public Task StartAsync(CancellationToken cancellationToken)
 
     public Task StartedAsync(CancellationToken cancellationToken)
     {
+        if (distributedApplicationOptions.DashboardEnabled && configuration["AppHost:BrowserToken"] is { Length: > 0 } browserToken)
```

Comment:
I thought that VS would still continue to look for the existing `Now listening on:` message. I'm assuming based on this that they've agreed to look for the extra message? This wouldn't work for `dotnet watch` for example?

Reply:
VS isn't looking for this. It is hard coding appending login to the URL. And it already knows the token.

Reply:
The question about `dotnet watch` is relevant though as that definitely watches console output to know when to launch the browser.

Reply:
It will continue to look for `Now listening on:`.